collector-configs/headsampling.yaml

```
@@ -5,7 +5,7 @@ receivers:
       http:
 
 processors:
-  # See: https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/probabilisticsamplerprocessor/README.md
+  # See: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/probabilisticsamplerprocessor
```

```suggestion
  # See: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/probabilisticsamplerprocessor#readme
```

Looks good as is!   One tiny nicety you can consider (or totally ignore!) would be to add the readme anchor